src/docs/data_usage_telemetry.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Data Usage and Telemetry
+---
+
+# Data Usage and Telemetry in Theia
+
+Eclipse Theia and Theia IDE are fully open-source projects. This means you can review the source code to verify exactly how the application behaves. Transparency is a core principle - no hidden services or undisclosed data collection take place.
+
+## No Built-in Telemetry
+
+The Theia IDE itself does not include built-in telemetry. It does not automatically collect or transmit user activity, project details, or code.
+
+However, tool builders who create custom products based on Theia can choose to add their own telemetry systems. If you are using a customized Theia-based product, you should consult that product's specific privacy policy and documentation for details about its data collection practices.
+
+## Automatic Network Connections
+
+While Theia IDE does not include telemetry, there are two automatic checks made during normal operation:
+
+- **Update Check for Theia IDE**: The IDE may query the download servers to see if a newer version of Theia IDE is available.
+- **Extension Update Check**: The IDE may query Open VSX to determine whether newer versions of installed extensions are available.
+
+These checks are limited to update metadata and do not transmit user code or workspace data.
```

Comment:
We may want to describe how to turn off these features for adopters and users within the Theia IDE and maybe even before they start the tool.

Reply:
We currently can't: https://github.com/eclipse-theia/theia-ide/issues/546